optaplanner-examples/src/main/java/org/optaplanner/examples/conferencescheduling/persistence/ConferenceSchedulingXlsxFileIO.java

```diff
@@ -1013,10 +1013,14 @@ private void writeTalkList() {
                 nextCell().setCellValue(talk.getFavoriteCount());
                 nextCell().setCellValue(talk.getCrowdControlRisk());
                 nextCell(talk.isPinnedByUser() ? pinnedStyle : defaultStyle).setCellValue(talk.isPinnedByUser());
-                nextCell().setCellValue(talk.getTimeslot() == null ? "" : DAY_FORMATTER.format(talk.getTimeslot().getDate()));
-                nextCell().setCellValue(talk.getTimeslot() == null ? "" : TIME_FORMATTER.format(talk.getTimeslot().getStartDateTime()));
-                nextCell().setCellValue(talk.getTimeslot() == null ? "" : TIME_FORMATTER.format(talk.getTimeslot().getEndDateTime()));
-                nextCell().setCellValue(talk.getRoom() == null ? "" : talk.getRoom().getName());
+                nextCell(talk.getTimeslot().equals(talk.getPublishedTimeslot()) ? defaultStyle : republishedStyle)
```

Comment:
@MusaTalluzi This caused an NPE when running the Generator ecause getTimeslot() returns null.
I've fixed it my branch by using java.util.Objects.equal() instead.